src/main/java/net/mcreator/ui/modgui/ToolGUI.java

```diff
@@ -274,8 +274,27 @@ public ToolGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		blocksAffected.setEnabled(false);
 
 		toolType.addActionListener(event -> {
-			if (toolType.getSelectedItem() != null)
-				blocksAffected.setEnabled(toolType.getSelectedItem().equals("Special"));
+			if (toolType.getSelectedItem() != null) {
```

Comment:
I don't think eg. shears use all types too, so consider extending this for other tool types. Also, move this to a method and call this method on edit mode too.